dashboard/app/controllers/code_reviews_controller.rb

```diff
@@ -31,9 +31,11 @@ def create
     code_review = CodeReview.new(
       user_id: current_user.id,
       project_id: project.id,
-      project_version: params[:version],
       script_id: params[:scriptId],
-      level_id: params[:levelId]
+      level_id: params[:levelId],
+      project_level_id: params[:levelId],  # TODO: send projectLevelId from the client
```

Comment:
If the level_id and project_level_id are different, we'll send them both right? Are you waiting for the client to send the separate value to reference it here?

Reply:
I think the client should always send both (even if they're the same).  Yes, I'm waiting for the change on the client to send it projectLevelId;